CodeSniffer/Standards/Squiz/Sniffs/WhiteSpace/MemberVarSpacingSniff.php

```diff
@@ -34,6 +34,16 @@ class Squiz_Sniffs_WhiteSpace_MemberVarSpacingSniff extends PHP_CodeSniffer_Stan
 
 
     /**
+     * The exception to the newline requirement on the first member var after class opener.
+     * false for newline requirement on the first member var after class opener
+     * true for no newline requirement on the first member var after class opener
+     *
+     * @var bool
+     */
+    public $firstVarException = false;
```

Comment:
Ah, I thought that you've wrapped existing code with an IF, which was disabled by default. Now I see that it's completely new code (maybe fixing part is copy-pasted from somewhere).